_includes/categorylinks.html

```diff
@@ -1,10 +1,12 @@
-    <div class="tablet:grid-col">
-      {% for link in include.links %}
-        {% if forloop.first == true %}
-          <h2 class = "usa-prose">{{ link.text }}</h2>
-          <a href = "{{ link.href }}">Overview</a><br />
-        {% else %}
-          <a href = "{{ link.href }}">{{ link.text }}</a><br />
-        {% endif %}
-      {% endfor %}
-    </div>
+<div class="tablet:grid-col">
+  {% for link in links %}
+    {%- if forloop.first == true -%}
+      <h2 class = "usa-prose">{{ link.title }}</h2>
```

Comment:
I know we looked at last week, but with fresh eyes it does seem like a lot of space before the heading, and I'd love to cut it down a little (go from 40 to 16). I think we may need to create a new css rule though to avoid fighting with specificity with `usa-prose`. I am thinking we add a class like `.heading-md` to our custom css file something like

```
.heading-md {
  include @u-font('sans', 'md');
  margin-top: units(2);
}
```

Reply:
I took a stab at it here: 23792fb
  
Let me know what you think. The specificity of `.usa-prose h2` was getting in the way, so I added a second rule for the margin-top

Reply:
Looking great!